quinn-proto/src/crypto/rustls.rs

```diff
@@ -399,3 +402,14 @@ impl crypto::PacketKey for PacketKey {
         }
     }
 }
+
+/// Cipher suites suitable for QUIC
+///
+/// The list is equivalent to TLS1.3 ciphers.
+/// Cipher suites are ordered by efficiency. On most desktop and mobile hardware
+/// AES ciphers suites will be more efficient due to hardware acceleration.
```

Comment:
Let's add a note here to remove once rustls is updated; I don't see value in us maintaining our own list long term.